lib/capybara/node/actions.rb

```diff
@@ -136,10 +127,10 @@ def choose(locator = nil, **options)
       # @overload check([locator], **options)
       #   @param [String] locator           Which check box to check
       #
-      #   @option options [String] option  Value of the checkbox to select
-      #   @option options [String, Regexp] id       Match fields that match the id attribute
-      #   @option options [String] name     Match fields that match the name attribute
-      #   @option options [String, Array<String>, Regexp] class    Match fields that match the class(es) provided
+      #   @option options [String] :option  Value of the checkbox to select
```

Comment:
Not really sure the change to symbols is necessary -- probably should be a separate PR so we can compare the changes it makes to the docs more easily

Reply:
Yes, I think so 👍